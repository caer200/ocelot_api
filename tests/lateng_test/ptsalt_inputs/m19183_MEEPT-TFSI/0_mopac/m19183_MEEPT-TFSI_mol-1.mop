CHARGES

 
H	8.736908 +1 13.074990 +1 12.950914 +1
C	8.717770 +1 13.556494 +1 13.769624 +1
C	9.517082 +1 14.714567 +1 13.914222 +1
C	7.914945 +1 13.116681 +1 14.792728 +1
C	9.456739 +1 15.387691 +1 15.159271 +1
N	10.315853 +1 15.149629 +1 12.863480 +1
H	7.375952 +1 12.345090 +1 14.663908 +1
C	7.875948 +1 13.785733 +1 16.023993 +1
C	8.643197 +1 14.908342 +1 16.199080 +1
S	10.423929 +1 16.760919 +1 15.542748 +1
C	11.047528 +1 16.330439 +1 12.871026 +1
C	10.515524 +1 14.200094 +1 11.735366 +1
H	7.324733 +1 13.465155 +1 16.728253 +1
H	8.624446 +1 15.366091 +1 17.031350 +1
C	11.151011 +1 17.141005 +1 14.029808 +1
C	11.724084 +1 16.767876 +1 11.709977 +1
H	10.556881 +1 13.280202 +1 12.098687 +1
H	11.387841 +1 14.393310 +1 11.308935 +1
C	9.431612 +1 14.255070 +1 10.672381 +1
C	11.921683 +1 18.311974 +1 14.016507 +1
H	11.652411 +1 16.264518 +1 10.907561 +1
C	12.484398 +1 17.912375 +1 11.727361 +1
H	9.485821 +1 13.449103 +1 10.100094 +1
H	8.541399 +1 14.266609 +1 11.105334 +1
O	9.588772 +1 15.418912 +1 9.873481 +1
H	11.980179 +1 18.845501 +1 14.800292 +1
C	12.592564 +1 18.695113 +1 12.881702 +1
H	12.945441 +1 18.177145 +1 10.940063 +1
C	8.741817 +1 15.330678 +1 8.728541 +1
H	13.124091 +1 19.482569 +1 12.880340 +1
H	7.815146 +1 15.133882 +1 9.015996 +1
H	9.047293 +1 14.590581 +1 8.146394 +1
C	8.766404 +1 16.616691 +1 7.959944 +1
H	9.702774 +1 16.902773 +1 7.813347 +1
H	8.339189 +1 16.488174 +1 7.076131 +1
O	8.071025 +1 17.607288 +1 8.689039 +1
C	7.953492 +1 18.824750 +1 7.974020 +1
H	7.497637 +1 19.488711 +1 8.532716 +1
H	7.434214 +1 18.674446 +1 7.156485 +1
H	8.845948 +1 19.153048 +1 7.738708 +1
