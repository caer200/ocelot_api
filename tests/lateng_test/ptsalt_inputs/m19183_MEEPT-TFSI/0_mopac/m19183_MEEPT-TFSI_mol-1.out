 *******************************************************************************
 ** Site#: 29200        For non-commercial use only    Version 19.090L 64BITS **
 *******************************************************************************
 ** Cite this program as: MOPAC2016, Version: 19.090L, James J. P. Stewart,   **
 ** Stewart Computational Chemistry, web: HTTP://OpenMOPAC.net. Days left: 174**
 *******************************************************************************
 **                                                                           **
 **                                MOPAC2016                                  **
 **                                                                           **
 *******************************************************************************

                              PM7 CALCULATION RESULTS

 *******************************************************************************
 *  CALCULATION DONE:                                Tue Oct  8 20:08:12 2019  *
 *  CHARGES    - IDENTIFY AND PRINT CHARGED ATOMS
 *  T=         - A TIME OF 172800.0 SECONDS REQUESTED
 *  DUMP=N     - RESTART FILE WRITTEN EVERY  7200.000 SECONDS
 *******************************************************************************
CHARGES


   ATOM   CHEMICAL          X               Y               Z
  NUMBER   SYMBOL      (ANGSTROMS)     (ANGSTROMS)     (ANGSTROMS)
 
     1       H          8.73690800  *  13.07499000  *  12.95091400  *
     2       C          8.71777000  *  13.55649400  *  13.76962400  *
     3       C          9.51708200  *  14.71456700  *  13.91422200  *
     4       C          7.91494500  *  13.11668100  *  14.79272800  *
     5       C          9.45673900  *  15.38769100  *  15.15927100  *
     6       N         10.31585300  *  15.14962900  *  12.86348000  *
     7       H          7.37595200  *  12.34509000  *  14.66390800  *
     8       C          7.87594800  *  13.78573300  *  16.02399300  *
     9       C          8.64319700  *  14.90834200  *  16.19908000  *
    10       S         10.42392900  *  16.76091900  *  15.54274800  *
    11       C         11.04752800  *  16.33043900  *  12.87102600  *
    12       C         10.51552400  *  14.20009400  *  11.73536600  *
    13       H          7.32473300  *  13.46515500  *  16.72825300  *
    14       H          8.62444600  *  15.36609100  *  17.03135000  *
    15       C         11.15101100  *  17.14100500  *  14.02980800  *
    16       C         11.72408400  *  16.76787600  *  11.70997700  *
    17       H         10.55688100  *  13.28020200  *  12.09868700  *
    18       H         11.38784100  *  14.39331000  *  11.30893500  *
    19       C          9.43161200  *  14.25507000  *  10.67238100  *
    20       C         11.92168300  *  18.31197400  *  14.01650700  *
    21       H         11.65241100  *  16.26451800  *  10.90756100  *
    22       C         12.48439800  *  17.91237500  *  11.72736100  *
    23       H          9.48582100  *  13.44910300  *  10.10009400  *
    24       H          8.54139900  *  14.26660900  *  11.10533400  *
    25       O          9.58877200  *  15.41891200  *   9.87348100  *
    26       H         11.98017900  *  18.84550100  *  14.80029200  *
    27       C         12.59256400  *  18.69511300  *  12.88170200  *
    28       H         12.94544100  *  18.17714500  *  10.94006300  *
    29       C          8.74181700  *  15.33067800  *   8.72854100  *
    30       H         13.12409100  *  19.48256900  *  12.88034000  *
    31       H          7.81514600  *  15.13388200  *   9.01599600  *
    32       H          9.04729300  *  14.59058100  *   8.14639400  *
    33       C          8.76640400  *  16.61669100  *   7.95994400  *
    34       H          9.70277400  *  16.90277300  *   7.81334700  *
    35       H          8.33918900  *  16.48817400  *   7.07613100  *
    36       O          8.07102500  *  17.60728800  *   8.68903900  *
    37       C          7.95349200  *  18.82475000  *   7.97402000  *
    38       H          7.49763700  *  19.48871100  *   8.53271600  *
    39       H          7.43421400  *  18.67444600  *   7.15648500  *
    40       H          8.84594800  *  19.15304800  *   7.73870800  *
 


          CARTESIAN COORDINATES 

    NO.       ATOM           X           Y           Z

     1         H          8.7369     13.0750     12.9509
     2         C          8.7178     13.5565     13.7696
     3         C          9.5171     14.7146     13.9142
     4         C          7.9149     13.1167     14.7927
     5         C          9.4567     15.3877     15.1593
     6         N         10.3159     15.1496     12.8635
     7         H          7.3760     12.3451     14.6639
     8         C          7.8759     13.7857     16.0240
     9         C          8.6432     14.9083     16.1991
    10         S         10.4239     16.7609     15.5427
    11         C         11.0475     16.3304     12.8710
    12         C         10.5155     14.2001     11.7354
    13         H          7.3247     13.4652     16.7283
    14         H          8.6244     15.3661     17.0313
    15         C         11.1510     17.1410     14.0298
    16         C         11.7241     16.7679     11.7100
    17         H         10.5569     13.2802     12.0987
    18         H         11.3878     14.3933     11.3089
    19         C          9.4316     14.2551     10.6724
    20         C         11.9217     18.3120     14.0165
    21         H         11.6524     16.2645     10.9076
    22         C         12.4844     17.9124     11.7274
    23         H          9.4858     13.4491     10.1001
    24         H          8.5414     14.2666     11.1053
    25         O          9.5888     15.4189      9.8735
    26         H         11.9802     18.8455     14.8003
    27         C         12.5926     18.6951     12.8817
    28         H         12.9454     18.1771     10.9401
    29         C          8.7418     15.3307      8.7285
    30         H         13.1241     19.4826     12.8803
    31         H          7.8151     15.1339      9.0160
    32         H          9.0473     14.5906      8.1464
    33         C          8.7664     16.6167      7.9599
    34         H          9.7028     16.9028      7.8133
    35         H          8.3392     16.4882      7.0761
    36         O          8.0710     17.6073      8.6890
    37         C          7.9535     18.8248      7.9740
    38         H          7.4976     19.4887      8.5327
    39         H          7.4342     18.6744      7.1565
    40         H          8.8459     19.1530      7.7387

 General Reference for PM7:
 "Optimization of Parameters for Semiempirical Methods VI: More Modifications to the 
 NDDO Approximations and Re-optimization of Parameters", J. J. P. Stewart, J. Mol. Mod., 1:32, 19 (2013)
 http://www.springerlink.com/openurl.asp?genre=article&id=doi:10.1007/s00894-012-1667-x

           Empirical Formula: C17 H19 N O2 S  =    40 atoms



      MOLECULAR POINT GROUP   :   C1  

          COMPUTED CHARGE ON SYSTEM: +0

   COMPUTED CHARGE ON SYSTEM = +0. THIS AGREES WITH THE CHARGE IN THE DATA-SET.

          JOB STOPPED HERE BECAUSE KEYWORD "CHARGES" WAS USED

 ******************************************************************************
 *                                                                            *
 *     Error and normal termination messages reported in this calculation     *
 *                                                                            *
 * JOB STOPPED HERE BECAUSE KEYWORD "CHARGES" WAS USED                        *
 * JOB ENDED NORMALLY                                                         *
 *                                                                            *
 ******************************************************************************



 TOTAL JOB TIME:             0.00 SECONDS

 == MOPAC DONE ==
